presto-hive/pom.xml

```diff
@@ -45,7 +45,6 @@
         <dependency>
             <groupId>io.prestosql.hadoop</groupId>
             <artifactId>hadoop-apache</artifactId>
-            <scope>provided</scope>
```

Comment:
remove from hive-hadoop2/pom.xml ?

Reply:
It is still needed there since the tests directly use Hadoop classes.